### ISSUE TITLE
Buttons in a level2 Toolbar Menu are duplicated in the map menu toolbar.

### ISSUE THREAD

**Issue by riverwanderer** [vassalengine/vassal#12179]
Buttons in a level2 Toolbar Menu are duplicated in the map menu toolbar (but working ok on module toolbar).

Observed in Vassal v3.7.0-beta2; but not in v3.6.14, nor in Vassal v3.6.15 (later has a different issue, residual to #[12093](https://github.com/vassalengine/vassal/pull/12128))

The issue:
Earlier versions of the C&C Nap module (e.g. [Version v4.1.8](https://vassalengine.org/wiki/File:CCNapoleonics4_1_8.vmod)), demonstrate the issue (run up a scenario; connect as Solitaire; choose deck; open hand window to see the error). In this case, buttons in their initial state (enabled) are duplicated and the duplicate is displayed on the toolbar thus:
<img width="1159" alt="image" src="https://user-images.githubusercontent.com/58135975/229719426-5a72bff0-8404-40ef-98da-5d5eb6ce4f30.png">

The difference between CCNap v4.1.8 and more recent versions is that the "Draw...xN" buttons have a disabling GP. In versions without the Disabling GP, the buttons work just fine under all versions of Vassal.

Note: A Disabling GP still exists on the parent Toolbar Menu ("Initial Draw") providing the locking functionality the module requires. That button is duplicating up to the main toolbar, but only once the Disabling takes effect, which may be the same or a related issue - see #12168 ).

**Comment by riverwanderer** [comment]
Was duplicate of #12208, fixed by #12262.
